cds-snc/c19-benefits-node
# People shouldn't be able to submit empty feedback

# Description

Add an error message when a user submits an empty feedback form: 

Will display the following with JS enabled:
![image](https://user-images.githubusercontent.com/87738/83565022-a58d6b80-a4eb-11ea-8e5c-c4b3f5f14658.png)

Will redirect to the following with JS Disabled: 
![image](https://user-images.githubusercontent.com/87738/83565070-bccc5900-a4eb-11ea-952a-7ec107e220ea.png)

## Test Instructions

1. Submit empty feedback with JS on. 
1. Error message should appear

1. Submit empty feedback with JS Disabled 
  - To disable open developer tools (cmd+shift+c_
  - Open settings finder (cmd+shift+p)
 - Type Javascript then select "disable Javascript"
 - Follow the same steps to enable it. 
1. Should redirect to error page 

## Help Requested

Please validate the text on the no-js page as we do not have a design for no javascript for successful and unsuccessful feedback submissions
